spring-cloud-stream-module-launcher/src/main/java/org/springframework/cloud/stream/module/launcher/ModuleLauncher.java

```diff
@@ -49,10 +49,7 @@
 public class ModuleLauncher {
```

Comment:
JavaDoc for this class should be updated, and most of what's here should move to ModuleLauncherApplication